cordcutter/__init__.py

```diff
@@ -76,7 +86,7 @@ async def tripped_breaker(self, command: ApplicationCommand) -> None:
         """
         logger.warning("🔌 Breaker tripped for %s!", command.qualified_name)
 
-        original_callback: Any = command.callback  # pyright: reportUnknownMemberType=false
+        original_callback: Any = command.callback  # pyright: ignore[reportUnknownMemberType=false]
```

Comment:
This looks to be out of scope of this PR?

Reply:
I'm not sure if it makes sense to do a pull request to add two `ignore[]`?